back/README.md

```diff
@@ -239,43 +239,6 @@ NOTE: Watch out that you don't accidently commit these changes!
 3. Comment out `asset_host` method (use original implementation from Carrierwave)
 
 
-## Creating Engines
```

Comment:
It is not clear to me why you took out this part of the readme. Isn't it valid anymore?

Reply:
I took it out because these steps are now explained on Notion: https://www.notion.so/citizenlab/Engines-and-features-5efffe90cb1e4b55ba5c3e01322d6a5a Do you think it's better to also keep them in the README?

Reply:
I think that this particular documentation is best kept in the code, because it is part of the open source repo. People outside CitizenLab doe not have access to Notion.

Reply:
I put back the explanation in the README file and made some changes (see also the discussion about licenses here: https://citizenlabco.slack.com/archives/C016C2EHURY/p1647599793014869?thread_ts=1647456720.891939&cid=C016C2EHURY)

Reply:
Is the link to Notion something that is allowed be in a file that can be read by everybody outside CitizenLab?

Reply:
Not, people outside Citizenlab wouldn't be able to view the notion page. I thought it could still be useful for developers from Citizenlab who would create an engine by following the instructions from the README (as the instructions in Notion have been added very recently) so they're aware of the Notion page. But if you don't think it makes sense to include the link, I'll remove it.

Reply:
Good point, I removed the link from the README.

Reply:
I forgot to push, now you can really see the changes.